app/controllers/simplified-account/settings/index.js

```diff
@@ -2,4 +2,5 @@ module.exports.index = require('./index.controller')
 module.exports.serviceName = require('./service-name/service-name.controller')
 module.exports.emailNotifications = require('./email-notifications/email-notifications.controller')
 module.exports.refundEmails = require('./email-notifications/refund-emails.controller')
+module.exports.paymentConfirmationEmails = require('./email-notifications/payment-confirmation-emails.controller')
```

Comment:
Consider exporting the sub settings controllers in the `email-notifications.controller`, then you only need to add the parent setting controller to this index.

It also makes the code more readable when creating routes (imo) e.g. 
```js
serviceSettingsController.emailNotifications.paymentConfirmationEmails.get
```